techniques/systemSettings/process/services/1.0/metadata.xml

```diff
@@ -0,0 +1,109 @@
+<!--
+Copyright 2017 Normation SAS
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, Version 3.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<TECHNIQUE name="Services technique">
+  <DESCRIPTION>Check and enforce the state of services</DESCRIPTION>
+
+  <MULTIINSTANCE>true</MULTIINSTANCE>
+  <BUNDLES>
+    <NAME>services</NAME>
+  </BUNDLES>
+
+  <TMLS>
+    <TML name="services"/>
+  </TMLS>
+
+  <TRACKINGVARIABLE>
+    <SAMESIZEAS>SERVICES_TECHNIQUE_NAME</SAMESIZEAS>
+  </TRACKINGVARIABLE>
+
+  <SECTIONS>
+    <SECTION name="Services posthook"  multivalued="false"  component="true" componentKey="SERVICES_TECHNIQUE_NAME"/>
+    <SECTION name="Services boot status"  multivalued="false"  component="true" componentKey="SERVICES_TECHNIQUE_NAME"/>
+    <SECTION name="Services status"  multivalued="true"  component="true" componentKey="SERVICES_TECHNIQUE_NAME">
+        <INPUT>
+            <NAME>SERVICES_TECHNIQUE_NAME</NAME>
+            <DESCRIPTION>Service name (as seen in chkconfig or systemctl, or Windows tasks)</DESCRIPTION>
+        </INPUT>
+     
+        <SELECT1>
+          <NAME>SERVICES_TECHNIQUE_STATUS</NAME>
+          <DESCRIPTION>Status</DESCRIPTION>
+          <ITEM>
+            <LABEL>Running</LABEL>
+            <VALUE>true</VALUE>
```

Comment:
Why not use the same value as the label ?
This would make the developer's (and reviewer's) life easier.